public/js/components/FormFields/FormFieldDateTextInput.js

```diff
@@ -0,0 +1,58 @@
+import React, {PropTypes} from 'react';
+import ShowErrors from '../Utilities/ShowErrors';
+import { errorPropType } from '../../constants/errorPropType';
+import moment from 'moment';
+
+export default class FormFieldDateTextInput extends React.Component {
+
+
+    static propTypes = {
+        fieldLabel: PropTypes.string,
+        fieldName: PropTypes.string,
+        fieldValue: PropTypes.number,
+        fieldErrors: PropTypes.arrayOf(errorPropType),
+        formRowClass: PropTypes.string,
+        onUpdateField: PropTypes.func
+    };
+
+    milisecondsToString = (ms) => {
+        moment(ms).format("yyyy-MM-dd");
+    }
+
+    state = {
+        fieldDisplayValue: this.milisecondsToString(this.props.fieldValue)
+    };
+
+    onUpdate = (e) => {
+        let momentDate = moment(e.target.value, "YYYY-MM-DD", true);
+        if (momentDate.isValid()) {
+            this.setState ({
+                fieldDisplayValue: e.target.value
+            });
+            this.props.onUpdateField(momentDate.valueOf());
+        }
+    }
+
+    componentWillReceiveProps(newProps) {
+        this.milisecondsToString(newProps.fieldValue);
+    }
+
+
+    render() {
+
+        return (
+            <div className={this.props.formRowClass || "form__row"}>
+                {this.props.fieldLabel ? <label htmlFor={this.props.fieldName} className="form__label">{this.props.fieldLabel}</label> : false}
+                <input
+                    type="date"
```

Comment:
Just for Posterity, this is behind a feature switch in FF<55 but falls back to a text field anyway.

Reply:
Yes! Rationale for going with this method is a combination of time already spent on this, complexity of task and just getting it ready for launch across newsroom, where majority use Chrome.

Reply:
Yeah this doesn't work at all in my version of FF, it's impossible to enter text :(